src/main/kotlin/lotto/controller/LottoGameController.kt

```diff
@@ -0,0 +1,32 @@
+package lotto.controller
+
+import lotto.domain.AutoLotto
+import lotto.domain.LottoGame
+import lotto.domain.LottoGameResult
+import lotto.view.InputView
+import lotto.view.ResultView
+
+class LottoGameController {
+
+    fun start() {
+        val purchaseResult = InputView.purchaseCost()
+        InputView.numberOfPurchase(purchaseResult.numberOfGames)
+
+        val gameBoard = LottoGame.pick(purchaseResult.numberOfGames, AutoLotto)
+        gameBoard.forEach { ResultView.printChosenNumber(it.numbers) }
+
+        val winnerNumber = InputView.winningNumberOfLastWeek()
+        val gameResults = LottoGame.getResultOfGames(gameBoard, winnerNumber.winnerNumber)
+        ResultView.noticeOfPrize()
+        val statistics = LottoGameResult.winningStatistics(gameResults)
+        statistics.forEach {
+            ResultView.printWinningStatistics(
+                it.key.criteriaForWinning,
+                it.key.prize,
+                it.value
+            )
+        }
```

Comment:
> winningStatistics에서 반환한 값을 psintWinningStatics 에 전달하여 느슨한 결합을 만들어보는건 어떨까요?

이 부분을 너무 추상적으로 말씀드린 것 같네요 :) 
`ResultView.printWinningStatistics`이 로또 결과를 출력하는 역할을 모두 가지고있는게 어떨까요?
```suggestion
        val statistics = LottoGameResult.winningStatistics(gameResults)
        ResultView.printWinningStatistics(statistics)
```

Reply:
```
statistics.forEach {
            ResultView.printWinningStatistics(
                it.key.criteriaForWinning,
                it.key.prize,
                it.value
            )
        }
```
이 부분을 `ResultView.printWinningStatistics(statistics)` 여기서 구현하라는 말씀이신거죠?